rust/libnewsboat/src/utils.rs

```diff
@@ -262,6 +264,16 @@ pub fn is_valid_attribute(attribute:  &str) -> bool {
     VALID_ATTRIBUTES.contains(&attribute)
 }
 
+pub fn strwidth( rs_str: &str) -> usize {
+    let control = rs_str.chars().fold( true, |acc, x| acc & !x.is_control());
+
+    if control {
+            return UnicodeWidthStr::width(rs_str);
```

Comment:
This is indented too far to the right.

Reply:
Thanks! good catch, vim has been indenting weird on rust lately.